package.json

```diff
@@ -74,6 +74,7 @@
     "express-session": "^1.15.1",
     "isomorphic-relay": "^0.7.4",
     "lodash": "^4.17.4",
+    "node-fetch": "^1.6.3",
```

Comment:
I just did `yarn add node-fetch` but there was no `yarn.lock` changes. Is that correct? I think I'm missing something in the flow of adding a new dep...